fred2/campaigntreeview.cpp

```diff
@@ -118,7 +118,7 @@ void campaign_tree_view::OnDraw(CDC* pDC)
 	Assert(r);
 
 	pDC->SelectObject(&black_pen);
-	// draw level seperators
+	// draw level separator
```

Comment:
This should be plural
```suggestion
	// draw level separators
```

Reply:
Ooops, good catch :-)